src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,5 +166,14 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    public static String getPrintableString(Printable...printables){
+        String printableString = "";
+        for(Printable next: printables){
+            printableString = printableString + next.getPrintableString();
```

Comment:
Be aware that there are a few ways to concatenate strings in Java, such as `StringBuilder`, `String.join`, etc.